articles/active-directory/saas-apps/betterworks-tutorial.md

```diff
@@ -147,4 +150,4 @@ When you click the BetterWorks tile in the Access Panel, you should be automatic
 
 - [What is conditional access in Azure Active Directory?](../conditional-access/overview.md)
 
-- [Try BetterWorks with Azure AD](https://aad.portal.azure.com/)
\ No newline at end of file
+- [Try BetterWorks with Azure AD](https://aad.portal.azure.com/)
```

Comment:
It's strange that it exists in the web but looks like doesn't exist in the master branch, after created the PR, this discrepancy happened.